hyperion/grid/voronoi_grid.py

```diff
@@ -450,10 +456,17 @@ def write(self, group, quantities='all', copy=True, absolute_paths=False, compre
         voronoi_table['volume'][np.isnan(voronoi_table['volume'])] = -1.
         voronoi_table['volume'][np.isinf(voronoi_table['volume'])] = -1.
 
-        if voronoi_table['neighbours'].dtype != np.int32:
-            raise TypeError("neighbours should be int32")
+        # Remove the dense neighbours from the table.
+        voronoi_table = voronoi_table['coordinates', 'volume', 'bb_min', 'bb_max']
```

Comment:
Yes, I think that would make sense. Is the sparse representation computed on-the-fly in the C++ code? (i.e. will we actually save memory if we get rid of the dense representation?)